pdal/GDALUtils.hpp

```diff
@@ -771,13 +771,21 @@ class PDAL_DLL Raster
                      coords);
         double minx = coords[0];
         double miny = coords[1];
+        BOX2D output(minx, miny, maxx, maxy);
+        return output;
+    }
+
+    BOX3D bounds(int nBand) const
+    {
+
+        BOX2D box2 = bounds();
 
         double minimum; double maximum;
         double mean; double stddev;
-        statistics(1, &minimum, &maximum, &mean, &stddev);
+        statistics(nBand, &minimum, &maximum, &mean, &stddev);
 
-        BOX3D output(minx, miny, minimum,
-                     maxx, maxy, maximum);
+        BOX3D output(box2.minx, box2.miny, minimum,
```

Comment:
Might be nice to have the following:

    explicit BOX3D(const BOX2D& box, double minz, double maxz) :
        BOX2D(box), minz(minz), maxz(maxz)
    {}